bin/capsule-certs-generate

```diff
@@ -45,11 +45,20 @@ CONFIG_CONTENT = <<EOF
 EOF
 
 config_file = temp_file('capsule-certs', CONFIG_CONTENT)
+
+# generate our parser cache and add it to the config file
+parser_cache = temp_file('capsule-certs.parser_cache', `kafo-export-params -c #{config_file.path} -f parsercache --no-parser-cache`)
```

Comment:
now i understand your roundabout question. yeah, this wouldn't work on a production machine because there would be no puppet strings to generate the parser cache. dunno why i didn't catch that. reworking.